Kajabi/sage-lib
# Transaction Card

## Description
https://kajabi.atlassian.net/browse/MAN-970

Adds new transaction card to the system.

### Screenshots
![Screen Shot 2021-02-04 at 11 59 36 AM](https://user-images.githubusercontent.com/1175111/106948397-80877300-66e0-11eb-8bf8-87bc15393348.png)

## Test notes


### Steps for testing
1. Visit http://localhost:4000/pages/breakout/object/transaction_card
2. Verify UI and docs
3. Verify Name and Product name truncate when viewed in smaller containers/screens

--

1. Visit http://localhost:6006/?path=/story/sage-transactioncard--default
2. Verify UI and props
3. Verify Name and Product name truncate when viewed in smaller containers/screens

## Related
- NA
